editor.planx.uk/src/@planx/components/MultipleFileUpload/Public.tsx

```diff
@@ -227,4 +234,55 @@ const InteractiveFileListItem = (props: FileListItemProps) => {
   );
 };
 
-export default Component;
\ No newline at end of file
+const TagsPerFileContainer = styled(Box)(({ theme }) => ({
+  marginBottom: theme.spacing(4),
+}));
+
+interface FileTaggingModalProps {
+  uploadedFiles: FileUploadSlot[];
+  setShowModal: (value: React.SetStateAction<boolean>) => void;
+}
+
+const FileTaggingModal = (props: FileTaggingModalProps) => {
+  return (
+    <Dialog
+      open
+      onClose={() => props.setShowModal(false)}
+      data-testid="file-tagging-dialog"
+      maxWidth="xl"
+      PaperProps={{
+        sx: {
+          borderRadius: 0,
+          borderTop: `10px solid ${DEFAULT_PRIMARY_COLOR}`,
```

Comment:
I think we'll eventually want to access/reference the team theme color here instead of `DEFAULT_PRIMARY_COLOR`